boards/xtensa/esp32/esp32.dts

```diff
@@ -24,11 +24,11 @@
 };
 
 &cpu0 {
-	clock-frequency = <ESP32_CLK_CPU_240M>;
+	clock-frequency = <ESP32_CLK_CPU_80M>;
```

Comment:
The commit message subject and body seem to be out of sync, and there's a lot of other changes in here that don't seem to be directly related to XIP support, building a bootloader, or disabling a WDT.  Like this one.

Could you break this down into more targeted commits?

Also the text about disabling the bootloader-enabled WDT seems to conflict with what's in the third commit, where the WDT is still enabled if that wasn't done by the bootloader (where some alias variables are moved).

Reply:
> Also the text about disabling the bootloader-enabled WDT seems to conflict with what's in the third commit, where the WDT is still enabled if that wasn't done by the bootloader (where some alias variables are moved).

Not so clear for me. Current code performs this:

1. If board starts **without**esp-idf bootloader, it disables watchdog timer used on 1st bootloader (rom).
2. If board starts **with** bootloader, it means 2nd bootloader will trigger RTC WDT to perform some checks. Then we disable that before zephyr starts.

Reply:
@pabigot I've pushed new changes based on all discussion. Let me know if this one is still valid.